tensorflow/lite/core/model_builder.cc

```diff
@@ -90,6 +98,114 @@ std::unique_ptr<FlatBufferModel> FlatBufferModel::VerifyAndBuildFromBuffer(
                                       error_reporter);
 }
 
+void FlatBufferModel::ByteSwapSerializedModel(std::string* serialized_model){
```

Comment:
I also want to add conditional building (`#ifdef FLATBUFFERS_LITTLEENDIAN`) for all swapping functions which are only needed for BE targets.

Reply:
Yes, that makes sense. I'll add the macro conditional statement for them. Thank you very much for all the valuable comments!

Reply:
Done.